src/panels/KaitoModelsPanel.ts

```diff
@@ -0,0 +1,295 @@
+import { BasePanel, PanelDataProvider } from "./BasePanel";
+import * as vscode from "vscode";
+import * as k8s from "vscode-kubernetes-tools-api";
+import { failed } from "../commands/utils/errorable";
+import { MessageHandler, MessageSink } from "../webview-contract/messaging";
+import { ToVsCodeMsgDef, ToWebViewMsgDef } from "../webview-contract/webviewDefinitions/kaitoModels";
+import { InitialState } from "../webview-contract/webviewDefinitions/kaitoModels";
+import { TelemetryDefinition } from "../webview-contract/webviewTypes";
+import { invokeKubectlCommand } from "../commands/utils/kubectl";
+import { tmpdir } from "os";
+import { writeFileSync } from "fs";
+import { join } from "path";
+// import { DeploymentState } from "../webview-contract/webviewDefinitions/periscope";
+
+export class KaitoModelsPanel extends BasePanel<"kaitoModels"> {
+    constructor(extensionUri: vscode.Uri) {
+        super(extensionUri, "kaitoModels", {
+            deploymentProgressUpdate: null,
+        });
+    }
+}
+// interface DeploymentProgressUpdate {
```

Comment:
💡 might be delete this or is there a eason for latter usage hence commented out?